examples/spark/build.gradle.kts

```diff
@@ -0,0 +1,7 @@
+description = "An Example GraphQL service servied by a Spark HTTP server"
+
```

Comment:
if you add `application` plugin and configure main class you should be able to run the app using `run` task. See https://docs.gradle.org/current/userguide/application_plugin.html

Reply:
:+1: 
Added an application entry in the spark `build.gradle.kts` file.

Reply:
There's a typo in line 1: "service servied by" should be "service *served* by"

Reply:
Updated